### PR TITLE
i18n

### PR DESCRIPTION
**Pull request by jarjan**
I added basic functionality for internationalization or shortly i18n. Translation string are located under `src/locales` folder. For now I only i18n-ed footer and header. And snapshot testing is broken, need to fix :(

- [x] Fix snapshot testing
- [x] Error page
- [x] Blog page
- [x] Contact page
- [x] Friends page
- [x] Team page
- [x] Gallery page
- [x] Home page
- [x] Partners page
- [x] Projects page
- [x] Subscribe page
- [x] Support page
- [x] Values page

### CODE REVIEW

File: src/locales/de.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "active_de": "bold",
+  "active_en": "normal",
+  "footer_contact": "Kontakt",
+  "footer_friends": "Freunde",
+  "footer_partners": "Partnern",
```
Comment (hikes101) [inline src/locales/de.json]: Small typo here, "Partner" would be correct

**Comment by pheeria** [comment]
This is really cool! 🥇 
Do you have an idea how we can avoid usage of tons of lines of translations in the single json and have them nicely split across multiple files?

File: src/views/partials/main/header.hbs [hunk 2]
```diff
@@ -41,6 +41,8 @@
               </span>
             </div>
           </div>
+          <a class="nav-link text-light font-weight-{{{__ 'active_de'}}}" href="/de">DE</a>
```
Comment (pheeria) [inline src/views/partials/main/header.hbs]: This looks like a haaaack 🤣 
It's ok for now, but let's keep it in mind to have a better solution!

File: src/index.js [hunk 3]
```diff
@@ -23,11 +27,36 @@ app.engine(
     extname: "hbs",
     defaultLayout: "main",
     partialsDir: `${__dirname}/views/partials/`,
-    layoutsDir: `${__dirname}/views/layouts/`
+    layoutsDir: `${__dirname}/views/layouts/`,
+    helpers: {
+      __: function() { return i18n.__.apply(this, arguments); },
+      __n: function() { return i18n.__n.apply(this, arguments); }
```
Comment (pheeria) [inline src/index.js]: As far as I understand from the docs, we will probably not need ```__n``` anytime soon. :)

**Comment by jarjan** [comment]
[i18n – internationalization](https://trello.com/c/AhDTiYUh/143-i18n-internationalization)

File: src/index.js (hunk 3 above)
Comment (jarjan) [reply src/index.js]: Yes, but let's keep it anyways. Maybe we will need in future.

File: src/views/partials/main/header.hbs (hunk 2 above)
Comment (jarjan) [reply src/views/partials/main/header.hbs]: If it works .... ¯\_(ツ)_/¯

**Comment by jarjan** [comment]
> This is really cool! 🥇
> Do you have an idea how we can avoid usage of tons of lines of translations in the single json and have them nicely split across multiple files?

Nope, usually all translations are done in one file per locale.

**Comment by pheeria** [comment]
Is this pr wip or do you want to merge and split it into separate tasks?

File: package.json [hunk 4]
```diff
@@ -56,7 +55,7 @@
   "devDependencies": {
     "browser-sync": "^2.26.7",
     "dotenv": "^8.1.0",
-    "eslint": "^6.5.1",
+    "eslint": "^5.16.0",
```
Comment (pheeria) [inline package.json]: Why downgrade?

Comment (jarjan) [reply package.json]: @pheeria oh, it was due to problems with `prettier-eslint`, but I found workaround to not use it and forgot to revert eslint version. Will do, thanks!

**Pull request merged**